Command.fs

```diff
@@ -126,7 +131,7 @@ module Types =
         { /// <summary>The original command.</summary>
           Cmd : Command
           /// <summary>The command's microcode instantiation.</summary>
-          Microcode : Microcode<CTyped<MarkedVar>, Sym<MarkedVar>> list list
+          Microcode : Microcode<CTyped<MarkedVar>, Sym<MarkedVar>> list
```

Comment:
This is because, previously, commands translated into a sequential composition of parallel compositions of microcode commands, the idea being that microcode commands inside a single command were always arranged in such a way that parallel composition semantics was ok.

One of the main changes here is eliminating this behaviour and making every microcode instruction sequentially composed.  In practice this has no disadvantages (we efficiently compress the number of intermediate variables) and a lot of advantages (simplifies the logic, and removes some array semantics oddities).